README.md

```diff
@@ -20,13 +20,49 @@ module.exports = {
       options: {
         accessToken: 'YOUR_TOKEN',
         version: 'draft',
+        localAssets: true, // Optional parameter to download the images to use with Gatsby Image Plugin
         languages: ['de', 'at'] // Optional parameter. Omission will retrieve all languages by default.
       }
     }
   ]
 }
 ```
 
+### With Gatsby's image
+
+You need to set the `localAssets` option to `true`. Here is an example of the usage:
```

Comment:
this `localAssets` is so confusing for me. What is localAsset?

Reply:
It's called localAssets, because the plugin will download all the assets from Storyblok to create these dynamic Gatsby images

Reply:
That's why it's an optional paramater. But dynamic gatbsy images only work like this with `gatsby-source-filesystem`, that's what makes the functionality of the plugin